docs/how-to-configure-new-repository.md

```diff
@@ -52,7 +52,8 @@ policies:
 3. Every repo MUST enforce the CNCF `EasyCLA` check under the
    `Require status checks to pass before merging` section. (if the check isn't
    showing, have a CLA manager [enable the check for the
-   repo](https://project.lfcla.com/#/project/a0941000002wBz4AAE/cla))
+   repo](https://project.lfcla.com/#/project/a0941000002wBz4AAE/cla)). Check
+   with the OpenTelemetry's [EasyCLA admins](../assets.md#user-content-opentelemetry-managed-assets:~:text=Easy%20CLA) 
```

Comment:
> `:~:text=Easy%20CLA`

Does this work? Doesn't seem to do anything: https://github.com/open-telemetry/community/blob/master/assets.md#user-content-opentelemetry-managed-assets:~:text=Easy%20CLA

Reply:
on chrome and firefox it should work. Now there is a confusion as it highlights the commit description:

![image](https://user-images.githubusercontent.com/9950081/99320492-99d6fc80-2820-11eb-8838-6886c53d4dc5.png)

Reply:
Example that works now without confusion: https://github.com/open-telemetry/community/blob/master/assets.md#user-content-opentelemetry-managed-assets:~:text=YouTube%20channel%20OpenTelemetry